hls4ml/templates/oneapi/firmware/nnet_utils/nnet_helpers.h

```diff
@@ -13,22 +13,28 @@
 namespace nnet {
 
 template <class srcType, class dest_pipe, size_t SIZE> void convert_data(sycl::queue &q, srcType *src) {
```

Comment:
We should discuss what happens with this function vs the new DMA versions of these.